libraries/rib-base/src/main/java/com/badoo/ribs/android/integrationpoint/IntegrationPoint.kt

```diff
@@ -130,7 +131,13 @@ abstract class IntegrationPoint(
     abstract fun handleUpNavigation()
 
     override fun show(dialog: Dialog<*>, onClose: () -> Unit) {
-        val context = rootViewHost?.context ?: return
+        val context = rootViewHost?.context
+        if (context == null) {
+            RIBs.errorHandler.handleFatalError(
```

Comment:
Just thinking maybe this is a recoverable fatal error? It's serious but maybe still a lesser problem than crashing the app on production.

Reply:
I have same thoughts, that is why I used `errorHandler` instead of just throwing exception. Will replace with `handleNonFatalError` then.